### PR TITLE
Correct data for qa dashboards from dev versions including load balancers

### PR DESCRIPTION
**Pull request by gvelez17**
Changes to the dev dashboards to adapt to qa, mainly boilerplate except the load balancers pulled from AWS

### CODE REVIEW

File: grafana/provisioning/dashboards/qa/anchorservice.json [hunk 1]
```diff
@@ -465,7 +465,7 @@
                     "alias": "",
                     "dimensions": {
                         "AvailabilityZone": "*",
-                        "LoadBalancer": "app/ceramic-dev-cas-ipfs-nd-alb-ex/190eb438ebba5068",
+                        "LoadBalancer": "app/ceramic-qa-cas-ipfs-nd-alb-ex/190eb438ebba5068",
```
Comment (v-stickykeys) [inline grafana/provisioning/dashboards/qa/anchorservice.json]: Does this lb id need to change?

File: grafana/provisioning/dashboards/qa/anchorservice.json [hunk 2]
```diff
@@ -484,7 +484,7 @@
                     "alias": "",
                     "dimensions": {
                         "AvailabilityZone": "*",
-                        "LoadBalancer": "app/ceramic-dev-cas-ipfs-nd-alb-in/5f1227f17a902fe2",
+                        "LoadBalancer": "app/ceramic-qa-cas-ipfs-nd-alb-in/5f1227f17a902fe2",
```
Comment (v-stickykeys) [inline grafana/provisioning/dashboards/qa/anchorservice.json]: Does this lb id need to change?

**Review by v-stickykeys — CHANGES_REQUESTED** [review]
Just 2 lines that I think need to be updated

**Comment by v-stickykeys** [comment]
Also need the pubsub metrics filters deployed to AWS for those dashboards to work properly. See https://github.com/3box/ceramic-utils-lambda/blob/dev/src/scripts/generate-metric-filters.sh

File: grafana/provisioning/dashboards/qa/anchorservice.json (hunk 2 above)
Comment (gvelez17) [reply grafana/provisioning/dashboards/qa/anchorservice.json]: aaah - missed two!  yes, fixed!

File: grafana/provisioning/dashboards/qa/anchorservice.json (hunk 1 above)
Comment (gvelez17) [reply grafana/provisioning/dashboards/qa/anchorservice.json]: yes!

**Review by v-stickykeys — APPROVED** [review]

**Comment by gvelez17** [comment]
@smrz2001 I am going to merge this now but just fyi this is the thing we need to replace entirely soon https://github.com/3box/ceramic-stats/pull/42/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R32

**Pull request merged**